atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -189,13 +195,47 @@ def __init__(self, tasker, workflow, platforms, build_kwargs,
         self.build_kwargs = build_kwargs
         self.osbs_client_config = osbs_client_config
         self.config_kwargs = config_kwargs or {}
+        self.find_cluster_retry_delay = find_cluster_retry_delay
+        self.max_cluster_fails = max_cluster_fails
+        self.koji_upload_dir = self.get_koji_upload_dir()
+        self.fs_task_id = self.get_fs_task_id()
+        self.release = self.get_release()
 
         if worker_build_image:
             self.log.warning('worker_build_image is deprecated, use config_kwargs instead')
             self.config_kwargs.setdefault('build_image', worker_build_image)
 
         self.worker_builds = []
 
+    class ClusterRetryContexts(object):
+        def __init__(self, clusters, max_cluster_fails):
+            self.clusters = list(clusters)
+            self.retry_at = {}
```

Comment:
Can we document the intent of these 4 variables? (for instance, the names `failed` vs `fails` doesn't help understanding)